content/data/moon/kaguyatc_dtms.md

```diff
@@ -15,18 +15,18 @@ These data can be accessed in three ways:
 
 ### Overview
-The [Kaguya [Terrain Camera ](kaguyatc.md)analysis-ready[ data](kaguyatc.md) documentation provides an overview of the mission and sensors that were used as inputs to derive these data. Readers are advised to read the overview section of that documentation that describes the mission orbits, data resolutions, and physical sensors. The Kaguya spacecraft captured over 220,000 Terrain Camera (TC) observations in stereoscopic viewing mode, resulting in overlapping, fore and aft pointing data collection. In sum, we estimate that at least 361,000 candidate stereopairs exist with resolutions, orbits, and view geometries suitable for stereophotogrammetric DTM generation. The Kaguya TC USGS Generated DTM data set includes over 145,000 DTMs that wrap the Moon globally from 70˚ north latitude to 70˚ south latitude. The process used to select stereopairs for processing is described in detail below. The DTMs in this data set seek to fulfill the following goals:
+The [Kaguya Terrain Camera analysis-ready data](kaguyatc.md) documentation provides an overview of the mission and sensors that were used as inputs to derive these data. Readers are advised to read the overview section of that documentation that describes the mission orbits, data resolutions, and physical sensors. The Kaguya spacecraft captured over 220,000 Terrain Camera (TC) observations in stereoscopic viewing mode, resulting in overlapping, fore and aft pointing data collection. In sum, we estimate that at least 361,000 candidate stereopairs exist with resolutions, orbits, and view geometries suitable for stereophotogrammetric DTM generation. The Kaguya TC USGS Generated DTM data set includes over 145,000 DTMs that wrap the Moon globally from 70˚ north latitude to 70˚ south latitude. The process used to select stereopairs for processing is described in detail below. The DTMs in this data set seek to fulfill the following goals:
 
 1. Exhibit improved Minimum Reliably Resolvable Feature (MRRF) better than the SLDEM2015 product. This improvement is a function of input data resolution and stereopair view geometry.
 2. Achieve near-global coverage while biasing towards the extended mission phase when the spacecraft orbit was lower and data resolution generally was improved.
 3. Create DTMs at least 0.5˚ latitude in total along-track size.
 4. Produce overlapping DTM coverage to support accuracy and precision analysis of the results.
 
-The DTMs in this data set are released using the standard three-times input image resolution rule of thumb (CITE KIRK), resulting in DTM resolutions between X and Y meters per pixel.
+The DTMs in this data set are released using the standard three-times input image resolution rule of thumb (CITE KIRK), resulting in DTM resolutions between 7 and 37 meters per pixel.
```

Comment:
Does this need to be converted to a citation before release?

Reply:
Yup... 👍